frontend/api/types/Fetch.ts

```diff
@@ -1,5 +1,8 @@
 export enum FetchUrl {
     LOGIN = "/backend/auth",
+    USERS = "/backend/users",
+    POST = "/api/posts",
```

Comment:
if you want to get list of posts, then name it 'posts'
so we can name it "post" in case of single post